### ISSUE TITLE
Class Summary Per Student Per Topic: Fixing mastery charts in Per topic per student report

### ISSUE THREAD

**Issue by IvonArroyo** [marshall62/mathspring#209]
Hi David,

These are the charts that have not been working for a while:

Example:

We were able to see these in the report per topic per student (matrix with colors). i am logged in as fowler/fowler. It is hard to bring the teacher tools to the classroom without these reports.

For each student and each topic:
http://rose.cs.umass.edu/woj/WoAdmin?action=AdminViewReport&teacherId=92&classId=1072&studId=31153&reportId=24&state=showReport&extraParam=18

Per student, across topics:
http://rose.cs.umass.edu/woj/WoAdmin?action=AdminViewReport&teacherId=92&classId=1072&studId=31156&reportId=25&state=showReport

Per topic, across students in the class:
http://rose.cs.umass.edu/woj/WoAdmin?action=AdminViewReport&teacherId=92&classId=1072&reportId=26&state=showReport&extraParam=17
--Ivon

**Comment by marshall62** [comment]
I've tracked down the bug which must have existed ever since we put our server on ROSE.  
The problem is that the chart-building library we use, jchart, does not work with the Java that is installed on rose.   I've checked to make sure we are using the latest version of jchart.  

I'm going to have to ask CSCF to look into the version of java that is installed on this machine and if its possible we may try installing some other java to see if jcharts will work.   Installing a different java has its risks in that something else in the software may stop working.   I may take some time to look into an alternative to jcharts to avoid this messing around with java.

This means that any report that shows a graphic line chart is out of commission until this is solved.  
The links to these reports are not exactly right on the main menu.  One would have to open a report and then start clicking on links inside the data before these line chart reports happen.  So I'm not sure how many teachers are using these kinds of reports or why they'd care to track mastery over time.   It seems like more of a report used for researchers.
